### PR TITLE
fix(web): fix hosting price display

### PR DESCRIPTION
**Pull request by varun257**
fix display of price while ordering new web hosting package

resolves MANAGER-3615

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js [hunk 1]
```diff
@@ -188,7 +188,7 @@ angular.module('App').controller(
     }
 
     getResumePrice(price) {
-      return price.value === 0 ? this.$translate.instant('price_free') : this.$translate.instant('price_ht_label', { price: price.text });
+      return price.value === 0 ? this.$translate.instant('price_free') : `${price.text} ${this.$translate.instant('price_ht_label')}`;
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js]: Could you replace the translation by `{{price}} HT` instead ?

Comment (varun257) [reply packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js]: Okay sure will do. I thought of using the translation which was already there :)

Comment (FredericEspiau) [reply packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js]: Just make sure it's not used elsewhere and that you are not breaking anything 😄

Comment (varun257) [reply packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js]: I tried it, the problem is the `"€"` that has to come between price` `and `HT`, even if i add a translation which will be only for `"HT"`, i will have to concatenate `price.text` and translation in the end

Comment (FredericEspiau) [reply packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js]: I don't get it, please tell me:
- what happens when you have a `{{price}} HT` translation ? Isn't the `€` already in the `price`text ?

Comment (varun257) [reply packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js]: Yes `€` is in the text but the problem is, its getting displayed as `&#8364;`. Its not getting sanitized.

Comment (varun257) [reply packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js]: This works! thanks

File: packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js [hunk 2]
```diff
@@ -188,7 +188,8 @@ angular.module('App').controller(
     }
 
     getResumePrice(price) {
-      return price.value === 0 ? this.$translate.instant('price_free') : `${price.text} ${this.$translate.instant('domain_hosting_price_ht_label')}`;
+      return price.value === 0 ? this.$translate.instant('price_free')
+        : this.$translate.instant('domain_hosting_price_ht_label', { price: price.text }, undefined, false, 'sceParameters');
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/domain/webhosting-enable/domain-enable-web-hosting.controller.js]: Maybe you could add a comment to explain what you are doing here :)

**Pull request merged**